mlflow/server/js/src/experiment-tracking/components/ExperimentListTable.tsx

```diff
@@ -70,6 +69,14 @@ const useExperimentsTableColumns = () => {
         id: 'lastModified',
         accessorFn: ({ lastUpdateTime }) => Utils.formatTimestamp(lastUpdateTime, intl),
       },
+      {
+        header: intl.formatMessage({
+          defaultMessage: 'Description',
+          description: 'Header for the description column in the experiments table',
+        }),
+        id: 'description',
+        accessorFn: ({ tags }) => tags?.find(({ key }) => key === 'mlflow.note.content')?.value,
```

Comment:
could we add a dash to indicate nulls? just to be consistent with other tables

```suggestion
        accessorFn: ({ tags }) => tags?.find(({ key }) => key === 'mlflow.note.content')?.value ?? '-',
```